src/tools/plan-generator.ts

```diff
@@ -79,9 +83,11 @@ export class PlanGenerator {
   async createHandleVariable(handle: Handle): Promise<string> {
     handle.type.maybeEnsureResolved();
     const valInit = `val ${this.handleVariableName(handle)} = `;
+    const handleRestrictedType = this.ingressValidation
```

Comment:
Adding this in the plan-generator means that you don't support policies in proto format of recipes, what's the thinking? Do you think it's not needed, if so why? Or should we do it separately?

Reply:
In the next PR, I added a TODO to: https://github.com/PolymerLabs/arcs/blob/3455f9e90d85f7980cc613f82dd9027789ab58a6/src/tools/recipe2plan.ts#L56
to pass IngressValidation object into the `encodePlansToProto` (so the good thing is, we're tracking it).

Technically, there is no place for the restricted type in the recipe handle itself, that's why i chose to pass this additional argument in the first place.  If i understand the current use case for the proto recipes, they shouldn't contain any ingress recipes, but it's probably best that even the intermediate stores comply with policies. I'll send a PR shortly.